nomad/grapl-core.nomad

```diff
@@ -1233,6 +1233,13 @@ job "grapl-core" {
         # Hardcoded, but makes little sense to pipe up through Pulumi
         PLUGIN_WORK_QUEUE_HEALTHCHECK_POLLING_INTERVAL_MS = 5000
 
+        KAFKA_BOOTSTRAP_SERVERS = var.kafka_bootstrap_servers
+        KAFKA_SASL_USERNAME     = var.kafka_credentials["plugin-work-queue"].sasl_username
+        KAFKA_SASL_PASSWORD     = var.kafka_credentials["plugin-work-queue"].sasl_password
+
+        KAFKA_PRODUCER_TOPIC_GENERATOR = "generated-graphs"
+        # KAFKA_PRODUCER_TOPIC_ANALYZER = "TODO"
```

Comment:
not in this PR, more like a "hey don't forget to do this eventually"

Reply:
nit: could these be named `GENERATOR_KAFKA_PRODUCER_TOPIC` and `ANALYZER_KAFKA_PRODUCER_TOPIC`? When I see `KAFKA_PRODUCER_TOPIC_GENERATOR` I think it's something responsible for generating kafka producer topics.